### PR TITLE
Create a pull request template

### PR DESCRIPTION
**Pull request by gouniLee**
I've added a pull request and issue template. 
(fyi. https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/ ) and TEMPLATE form is from 'enactjs' repo.
When making a PR, we missed some steps such as writing release notes, and missing test some platforms. If we could see `checklist` before doing PR, it might help to fill the hole.

https://github.com/iLib-js/iLib/community

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### Checklist
+
+* [ ] At least one test case is included for this feature or bug fix
+* [ ] `ReleaseNotes` has added or is not needed
+* [ ] This PR has passed all of the test cases on `Nodejs` and `Browser
```
Comment (ehoogerbeets) [inline .github/PULL_REQUEST_TEMPLATE.md]: missing close quote
also, specify which browser exactly

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* [ ] This PR has passed all of the test cases on `QT/QML`
+* [ ] This is an API breaking change
```
Comment (ehoogerbeets) [inline .github/PULL_REQUEST_TEMPLATE.md]: Put "Requires a major version change."

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+### Description
+[//]: # (Describe the bug.  Explain what happened and what you expected to happen.)
+
+### Environment
+[//]: # (Describe the relevant environment, e.g. node/npm version, browser version.)
+
+
+### Reproduction Code
+[//]: # (Provide source code that demonstrates the bug.)
```
Comment (ehoogerbeets) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: ... and/or exact steps to reproduce.

File: .github/ISSUE_TEMPLATE/FEATURE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+about: Request a new feature or modification of an existing one
+
+---
+
+### Category
+
+[ ] New component
+[ ] New feature for existing component
+[ ] Modification of existing component or feature
+[ ] Other
```
Comment (ehoogerbeets) [inline .github/ISSUE_TEMPLATE/FEATURE.md]: New locale?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (gouniLee) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would put 'Chrome Browser'

**Pull request merged**